zerver/lib/home.py

```diff
@@ -180,7 +180,6 @@ def build_page_params_for_home_page_load(
 
     furthest_read_time = get_furthest_read_time(user_profile)
     two_fa_enabled = settings.TWO_FACTOR_AUTHENTICATION_ENABLED and user_profile is not None
-    billing_info = get_billing_info(user_profile)
```

Comment:
I think you can delete `get_billing_info` and its tests as well, since it's now unused?